app/views/shared/nav/_administration.html.haml

```diff
@@ -0,0 +1,7 @@
+- if !acting_as? && current_user.try(:administrator?)
```

Comment:
This partial gets included everywhere. `layouts/application` renders `shared/nav`, which then renders `shared/nav/administration`.

After further refactoring using the `NavTab` presenters (not in this branch), it's reduced down to 2 lines and probably not worth its own partial.